/bin/bash: /data/users2/bthapaliya/anaconda-main/anaconda3/lib/libtinfo.so.6: no version information available (required by /bin/bash)
arctrdgn002
Traceback (most recent call last):
  File "/data/users2/bthapaliya/DistributedFLExperiments/DistributedFL/fedml_experiments/standalone/subavg/main_subavg.py", line 221, in <module>
    subfedavgAPI.train()
  File "/data/users2/bthapaliya/DistributedFLExperiments/DistributedFL/fedml_api/standalone/subavg/subavg_api.py", line 66, in train
    new_mask, w_client,training_flops, num_comm_params = client.train(copy.deepcopy(w_global),
  File "/data/users2/bthapaliya/DistributedFLExperiments/DistributedFL/fedml_api/standalone/subavg/client.py", line 38, in train
    w_per  = real_prune(w_global,masks)
  File "/data/users2/bthapaliya/DistributedFLExperiments/DistributedFL/fedml_api/standalone/subavg/prune_func.py", line 48, in real_prune
    new_param_dict[name] = tensor * mask[name].to(weight_dev)
KeyError: 'bn1.running_mean'
